third-party/scripts/init.sh

```diff
@@ -6,10 +6,10 @@ echo "*** Initializing WASM build environment"
 
 if [ -z $CI_PROJECT_NAME ] ; then
    rustup update nightly
-   rustup update stable
 fi
 
 rustup target add wasm32-unknown-unknown --toolchain nightly
+rustup default nightly
```

Comment:
In this case node bill be build with nightly rust, right?

Reply:
I fixed a ↑ comment. So no need to use a nightly build. I fix.